test/library/standard/Memory/Initialization/setRTTypeBaseline.chpl

```diff
@@ -0,0 +1,10 @@
+use Set;
+
+var sa = new set([0..2] int);
+//var sd: set(domain(1));
```

Comment:
Did you mean to commit the commented out variation here?

Reply:
Sssortof.  I'd hoped to come back and find it was fixed after some other adjustments, and so uncomment it but had forgotten that domains don't support hashing yet (https://github.com/chapel-lang/chapel/issues/11340).  Ultimately, I'd prefer that the test include this case (and it would need to change to the `new set` form too in order to do so), so don't feel strongly about leaving it in commented or taking it out.  Do you have a preference?

Reply:
I'd probably recommend either putting the commented-out code all at the bottom explaining when it should be uncommented -- or leaving it out entirely and adding it in future work.